bundles/EcommerceFrameworkBundle/Resources/config/services.yaml

```diff
@@ -10,8 +10,10 @@ services:
 
     Pimcore\Bundle\EcommerceFrameworkBundle\Tools\SessionConfigurator:
         public: false
-        tags:
-            - { name: pimcore.session.configurator }
+        deprecated:
```

Comment:
`SessionConfigurator` extends from new `SessionBagListener`. Tags are removed in the core implementations otherwise they will trigger the deprecation warning here https://github.com/pimcore/pimcore/pull/12294/files#diff-461ad6908b3c6404bf8f5e543677e341e288d786e9038e8af7998fe82285026aR69-R72